### ISSUE TITLE
SES6 (devel) deployment broken

### ISSUE THREAD

**Issue by p-se** [SUSE/sesdev#575]
The command
```bash
sesdev create ses6
```
leads to
```bash
    master: ++ zypper --non-interactive install vim git-core iputils jq make iptables patch man command-not-found bc apparmor-utils apparmor-parser chrony hostname systemd-coredump
node2: Loading repository data...
    node2: Reading installed packages...
    node2: 'vim' is already installed.
    node2: No update candidate for 'vim-8.0.1568-5.11.1.x86_64'. The highest available version is already installed.
    node2: 'command-not-found' is already installed.
    node2: No update candidate for 'command-not-found-0.2.2+20190613.e6c2668-6.3.2.noarch'. The highest available version is already installed.
    node2: 'hostname' is already installed.
    node2: No update candidate for 'hostname-3.16-2.22.x86_64'. The highest available version is already installed.
    node2: 'iputils' is already installed.
    node2: No update candidate for 'iputils-s20161105-8.3.1.x86_64'. The highest available version is already installed.
    node2: Resolving package dependencies...
    node2: Problem: systemd-coredump-234-24.64.1.x86_64 requires systemd = 234-24.64.1, but this requirement cannot be provided
    node2:   not installable providers: systemd-234-24.64.1.x86_64[update]
    node2:  Solution 1: Following actions will be done:
    node2:   downgrade of systemd-234-24.72.1.x86_64 to systemd-234-24.64.1.x86_64
    node2:   downgrade of systemd-sysvinit-234-24.72.1.x86_64 to systemd-sysvinit-234-24.64.1.x86_64
    node2:   downgrade of udev-234-24.72.1.x86_64 to udev-234-24.64.1.x86_64
    node2:  Solution 2: do not install systemd-coredump-234-24.64.1.x86_64
    node2:  Solution 3: break systemd-coredump-234-24.64.1.x86_64 by ignoring some of its dependencies
    node2: 
    node2: Choose from above solutions by number or cancel [1/2/3/c/d/?] (c): c
    node2: Error in sesdev deployment script trapped!
    node2: +++ err_report 203
    node2: +++ set +x
    node2: => hostname: node2
    node2: => script: /tmp/vagrant-shell
    node2: => line number: 203
    node2: Bailing out!
    master: Loading repository data...
    master: Reading installed packages...
    master: 'vim' is already installed.
    master: No update candidate for 'vim-8.0.1568-5.11.1.x86_64'. The highest available version is already installed.
    master: 'command-not-found' is already installed.
    master: No update candidate for 'command-not-found-0.2.2+20190613.e6c2668-6.3.2.noarch'. The highest available version is already installed.
    master: 'hostname' is already installed.
    master: No update candidate for 'hostname-3.16-2.22.x86_64'. The highest available version is already installed.
    master: 'iputils' is already installed.
    master: No update candidate for 'iputils-s20161105-8.3.1.x86_64'. The highest available version is already installed.
    master: Resolving package dependencies...
    master: Problem: systemd-coredump-234-24.64.1.x86_64 requires systemd = 234-24.64.1, but this requirement cannot be provided
    master:   not installable providers: systemd-234-24.64.1.x86_64[update]
    master:  Solution 1: Following actions will be done:
    master:   downgrade of systemd-234-24.72.1.x86_64 to systemd-234-24.64.1.x86_64
    master:   downgrade of systemd-sysvinit-234-24.72.1.x86_64 to systemd-sysvinit-234-24.64.1.x86_64
    master:   downgrade of udev-234-24.72.1.x86_64 to udev-234-24.64.1.x86_64
    master:  Solution 2: do not install systemd-coredump-234-24.64.1.x86_64
    master:  Solution 3: break systemd-coredump-234-24.64.1.x86_64 by ignoring some of its dependencies
    master: Choose from above solutions by number or cancel [1/2/3/c/d/?] (c): 
    master: c
    master: Error in sesdev deployment script trapped!
    master: +++ err_report 203
    master: +++ set +x
    master: => hostname: master
    master: => script: /tmp/vagrant-shell
    master: => line number: 203
    master: Bailing out!
=== Deployment Finished ===

You can login into the cluster with:

  $ sesdev ssh ses6

Or, access the Ceph Dashboard with:

  $ sesdev tunnel ses6 dashboard
```

**Comment by smithfarm** [comment]
So, this is the relevant part of the error message:

```
    node2: Problem: systemd-coredump-234-24.64.1.x86_64 requires systemd = 234-24.64.1, but this requirement cannot be provided
    node2:   not installable providers: systemd-234-24.64.1.x86_64[update]
    node2:  Solution 1: Following actions will be done:
    node2:   downgrade of systemd-234-24.72.1.x86_64 to systemd-234-24.64.1.x86_64
```

Here is what is happening: the Vagrant image has a newer version of systemd (234-24.72.1) installed, than is available in SLE-15-SP1 Update channel (234-24.64.1). This is a known problem with Kiwi (the tool used to build the SLE-15-SP1 image): it **should** be taking packages from channels, but it only knows how to take packages from codestreams.

**Comment by smithfarm** [comment]
Minimal reproducer:

    sesdev create ses6 --single-node --no-provision
